README.md

```diff
@@ -50,6 +50,15 @@ their [official documentation](https://github.com/CycloneDX/cyclonedx-property-t
             SBOM.</td>
         <td><code>metadata</code></td>
     </tr>
+    <tr style="vertical-align: top;">
+        <td><b>siemens:sbomNature</b></td>
+        <td>Used to indicate the nature of the entire SBOM document. Possible values are:
+            <ul><li><code>binary</code> – The SBOM contains binary components.</li>
+                <li><code>source</code> – The SBOM contains source components.</li>
+            </ul>
+            A binary component may have source information attached, and vice versa.</td>
```

Comment:
Can we give a bit more context here? Something like this:

```suggestion
            This property is only relevant for package ecosystems that have this distinction, like [Debian](https://wiki.debian.org/Packaging/BinaryPackage) or [RPM](https://rpm-packaging-guide.github.io/#building-rpms).
 
            A binary component may have source information attached, and vice versa.</td>
```

Reply:
Ok! I took your text, exchanging only `only` for `mostly` and converting to HTML. Thanks!